docs/STANDALONE_MODE.md

```diff
@@ -0,0 +1,30 @@
+# Standalone mode
+
+Standalone mode enables plugin launching on different machine than Snap daemon (`snapteld`).
+
+To run plugin in standalone mode it is needed to launch plugin with `--stand-alone` flag, e.g.:
+```
+$ ./snap-plugin-collector-psutil --stand-alone
+```
+Plugin in standalone mode creates HTTP server for communication with Snap framework, by default plugin listens on `8182` port.
+
+It is possible to configure different port for communication with Snap framework using `--stand-alone-port` flag, e.g.:
+
+```
+$ ./snap-plugin-collector-psutil --stand-alone --stand-alone-port 8183
+```
+
+To load plugin in standalone mode it is required to provide URL to indicate a machine
```

Comment:
I think this section should have its own header so that it stands out more.

Reply:
@kjlyon Do you have any proposition of the header?

Reply:
Something like `Loading a plugin in snapteld` or simply, `Loading a plugin`